applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -385,16 +385,21 @@ static void cell_pos_req_wk(struct k_work *work)
 
 	ARG_UNUSED(work);
 
-	if (cell_pos_type == CELL_POS_TYPE_SINGLE) {
-		err = nrf_cloud_cell_pos_request(NULL, true, NULL);
+	if (cell_pos_type == LOCATION_TYPE_SINGLE_CELL) {
+		struct lte_lc_cells_info scell_inf = {.ncells_count = 0};
+
+		/* Obtain the single cell info from the modem */
+		err = nrf_cloud_location_scell_data_get(&scell_inf.current_cell);
```

Comment:
Error code from this call will be ignored in the next line. Is this intentional? If yes, it would probably be better to not set the return value to `err` in this line.

Reply:
not intentional, will add error check.